css/src/components/progress-bar.scss

```
@@ -0,0 +1,136 @@
+$progress-bar-background-color: $secondary-background-glow-high-contrast !default;
+$progress-bar-value-background-color: $primary !default;
+$progress-bar-value-background-color-warning: $warning !default;
+$progress-bar-value-background-color-danger: $danger !default;
+$progress-bar-value-background-color-success: $success !default;
+$progress-bar-height: $spacer-2 !default;
+$progress-bar-height-sm: $spacer-1 !default;
+$progress-bar-border-radius: $border-radius-rounded !default;
+$progress-bar-indeterminate-animation-duration: 2s !default;
+
+.progress-bar {
+	display: block;
+	position: relative;
+	width: 100%;
+	max-width: 100%;
+	height: $progress-bar-height;
+	padding: 0;
+	border: none;
+	border-radius: $progress-bar-border-radius;
+	background-color: $progress-bar-background-color;
+	appearance: none;
+	overflow: hidden;
+
+	&::-webkit-progress-bar {
+		background-color: $progress-bar-background-color;
+	}
+
+	&::-webkit-progress-value {
+		background-color: $progress-bar-value-background-color;
+	}
+
+	&::-moz-progress-bar {
+		background-color: $progress-bar-value-background-color;
+	}
```

Please feel free to disregard this - it's completely optional - but I wondered if it might be nice to group like background setting together.

```suggestion
.progress-bar {
	display: block;
	position: relative;
	width: 100%;
	max-width: 100%;
	height: $progress-bar-height;
	padding: 0;
	border: none;
	border-radius: $progress-bar-border-radius;
	appearance: none;
	overflow: hidden;

	&, &::-webkit-progress-bar {
		background-color: $progress-bar-background-color;
	}

	&::-webkit-progress-value,
	&::-moz-progress-bar {
		background-color: $progress-bar-value-background-color;
	}
```

---

Updated